docs/development/development.md

```diff
@@ -80,26 +100,39 @@ To locally run the controller as part of a k8s cluster, follow these steps:
     ```sh
     cd akri/discovery-handler-modules/onvif-discovery-handler/
-    sudo -s 
-    RUST_LOG=info DISCOVERY_HANDLERS_DIRECTORY=~/tmp/akri AGENT_NODE_NAME=myNode cargo run
+    sudo -E RUST_LOG=info DISCOVERY_HANDLERS_DIRECTORY=~/tmp/akri AGENT_NODE_NAME=myNode $HOME/.cargo/bin/cargo run
     ```
     To run the [debug echo Discovery Handler](#testing-with-debug-echo-discovery-handler), an environment variable,
     `DEBUG_ECHO_INSTANCES_SHARED`, must be set to specify whether it should register with the Agent as discovering
     shared or unshared devices. Run the debug echo Discovery Handler to discover mock unshared devices like so:
     ```sh
     cd akri/discovery-handler-modules/debug-echo-discovery-handler/
-    RUST_LOG=info DEBUG_ECHO_INSTANCES_SHARED=false DISCOVERY_HANDLERS_DIRECTORY=~/tmp/akri AGENT_NODE_NAME=myNode cargo run
+    RUST_LOG=info DEBUG_ECHO_INSTANCES_SHARED=false DISCOVERY_HANDLERS_DIRECTORY=~/tmp/akri AGENT_NODE_NAME=myNode $HOME/.cargo/bin/cargo run
     ```
 
-### To build containers
+## Building Akri's Containers
 `Makefile` has been created to help with the more complicated task of building the Akri components and containers for the various supported platforms.
 
-#### Establish a container repository
+### Tools for building Akri's Rust containers
+In order to cross-build Akri's Rust code for both ARM and x64 containers, several tools are leveraged.
+- The [Cross](https://github.com/rust-embedded/cross) tool can be installed with this command: `cargo install cross`.
+- `qemu` can be installed with:
+  ```sh
+  sudo apt-get install -y qemu qemu qemu-system-misc qemu-user-static qemu-user binfmt-support
+  ```
+
+  For `qemu` to be fully configured on Ubuntu 18.04, after running apt-get install, run these commands:
+  ```sh
+    sudo mkdir -p /lib/binfmt.d
+    sudo sh -c 'echo :qemu-arm:M::\\x7fELF\\x01\\x01\\x01\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x02\\x00\\x28\\x00:\\xff\\xff\\xff\\xff\\xff\\xff\\xff\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xfe\\xff\\xff\\xff:/usr/bin/qemu-arm-static:F > /lib/binfmt.d/qemu-arm-static.conf'
+    sudo sh -c 'echo :qemu-aarch64:M::\\x7fELF\\x02\\x01\\x01\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x02\\x00\\xb7\\x00:\\xff\\xff\\xff\\xff\\xff\\xff\\xff\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xfe\\xff\\xff\\xff:/usr/bin/qemu-aarch64-static:F > /lib/binfmt.d/qemu-aarch64-static.conf'
+    sudo systemctl restart systemd-binfmt.service
+  ```
+
+### Establish a container repository
 Containers for Akri are currently hosted in `ghcr.io/deislabs/akri` using the new [GitHub container registry](https://github.blog/2020-09-01-introducing-github-container-registry/). Any container repository can be used for private containers. If you want to enable GHCR, you can follow the [getting started guide](https://docs.github.com/en/free-pro-team@latest/packages/getting-started-with-github-container-registry).
```

Comment:
The getting started link is no longer valid

Reply:
the link seems to be working for me but i could see how it is not quite specific enough of a location

Reply:
Changed it to more directly point to the info on creating the container registry